dev/cloudformation/grid-dev-core.yml

```diff
@@ -195,6 +196,20 @@ Resources:
         ReadCapacityUnits: 1
         WriteCapacityUnits: 1
 
+  UploadStatusTable:
+    Type: AWS::DynamoDB::Table
+    Properties:
+      TableName: UploadStatusTable
+      AttributeDefinitions:
+      - AttributeName: id
+        AttributeType: S
+      KeySchema:
+      - AttributeName: id
+        KeyType: HASH
+      ProvisionedThroughput:
+        ReadCapacityUnits: 1
+        WriteCapacityUnits: 1
```

Comment:
This table should probably have a TTL of a few minutes to avoid us having to add code to tidy up rows.